spark/src/main/scala/io/spark/ddf/etl/TransformationHandler.scala

```diff
@@ -96,26 +101,20 @@ class TransformationHandler(mDDF: DDF) extends CoreTransformationHandler(mDDF) {
     val rMapped = dfrdd.map {
```

Comment:
Resolved by
val partdfres: REXP = rsess.eval(dfvarname)

@Huandao0812 can you help me understand why I have to enforce the type? According the the function I shouldn't need to: https://github.com/yannrichet/rsession/blob/e728223b7bc38b0d737f978772ef0929d0b759f4/Rsession/src/org/math/R/Rsession.java#L1170